source/_docs/guides/drupal-8-composer-no-ci.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Drupal 8 and Composer on Pantheon Without Continuous Integration
+description: Learn how to manage Drupal 8 using Composer with Pantheon.
+tags: [moreguides]
+categories: [drupal]
+type: guide
+permalink: docs/guides/:basename/
+contributors:
+  - ataylorme
+  - dwayne
+  - davidneedham
+---
+
+In an ideal situation, you might ignore the vendor directory and use [an automated build step with continuous integration](https://pantheon.io/docs/guides/build-tools/) to install your site with composer on the fly. But sometimes your project doesn’t have the budget or time needed to justify this workflow.
```

Comment:
I think this paragraph would benefit from a little more exposition. It starts with assuming knowledge of a vendor directory and Composer. Do we need to explain what those are?

Reply:
Perhaps we remove the first paragraph altogether and rework the second paragraph to be the into.

> In this guide, we’re going to run through the bare necessities to use [Composer](https://getcomposer.org/) for managing a Drupal 8 site on your local machine and pushing to Pantheon. 

We talk about the `vendor` directory later with

> **Note** the `vendor` directory is being committed to Pantheon. This is because Pantheon needs the full site artifact. If you prefer to ignore the `vendor` directory then take a look at our build tools guide for documentation on the more advanced automated workflow with a build step.

Reply:
I agree. That is much clearer.